src/BulkTools/HTTPBulkToolsResponse.php

```diff
@@ -362,18 +360,4 @@ protected function outputBody()
         $this->createBody();
         parent::outputBody();
     }
-
-    /**
-     * Catches fatal PHP error and output something useful for the front end
-     */
-    public function shutdown()
```

Comment:
Technically, removing this method breaks semver. Do we want to keep an empty method and mark it has deprecated?

Reply:
Opps, merged a bit soon, yes this is worth keeping just for semver - I'll pop it back in

Reply:
Yup, I've added back in an empty function